pkg/metrics/metrics.go

```diff
@@ -194,6 +197,16 @@ var (
 		ConstLabels: prometheus.Labels{"source": LabelEventSourceAPI},
 	})
 
+	LabelScope = "scope" // @TODO to delete when PR-5296 is merged.
+
+	// KVStore statistics
+
+	// KVStoreUsageTotal are the number of iteractions in kvstore per subsystem and type
+	KVStoreUsageTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "kvstore_usage_total",
```

Comment:
kvstore_operations_total might be better here, since that's what we're counting directly (I think?). Interactions isn't wrong, just unexpected.